### PR TITLE
Updated Spanish translation

### PR DESCRIPTION
**Pull request by BackpackXl**

### CODE REVIEW

**Comment by rafaelvcaetano** [comment]
Sorry about leaving this here for so long! I didn't want to give the work of constantly opening new PRs after every new string I added. But I think that all text changes for the next release are done. Or at least, I will try to avoid changing anything in that regard. If you still want to update the strings, I think now is the perfect time! And thanks for your effort!

**Review by agatemosu — COMMENTED** [review]
this is untranslated

File: app/src/main/res/values-b+es/strings.xml [hunk 1]
```diff
@@ -34,7 +36,10 @@
     <string name="error_rom_not_found">No se pudo encontrar ROM</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:36]:
```suggestion
    <string name="error_rom_not_found">No se pudo encontrar la ROM</string>
```

**Comment by BackpackXl** [comment]
Thank you for your response, now the translation is ready and looks decent.

File: app/src/main/res/values-b+es/strings.xml [hunk 2]
```diff
@@ -34,7 +36,10 @@
...
     <string name="error_rom_not_found_info">No se ha encontrado la ROM seleccionada. Asegúrese de que la ROM seleccionada se encuentre en el directorio de búsqueda de la ROM actual.</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:37]:
```suggestion
    <string name="error_rom_not_found_info">No se ha encontrado la ROM seleccionada. Asegúrate de que la ROM seleccionada se encuentre en el directorio de búsqueda de ROM actual.\n%1$s</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 3]
```diff
@@ -78,9 +84,13 @@
     <string name="cant_load_empty_slot">No se puede cargar un espacio vacío</string>
     <string name="failed_reset_emulation">No se pudo restablecer</string>
     <string name="save_states_not_supported">Los estados de guardado no son compatibles mientras se ejecuta el firmware</string>
+    <string name="save_states_unavailable_ra_hardcore_enabled">Guardados no disponibles en modo Hardcore</string>
     <string name="rewind_time_seconds">%1$ss</string> <!-- Ex: 43.56s -->
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:88]:
```suggestion
    <string name="rewind_time_seconds">%1$s s</string> <!-- Ex: 43.56 s -->
```

File: app/src/main/res/values-b+es/strings.xml [hunk 4]
```diff
@@ -78,9 +84,13 @@
...
     <string name="rewind_time_minutes_seconds">%1$dm%2$ss</string> <!-- Ex: 2m37.93s -->
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:89]:
```suggestion
    <string name="rewind_time_minutes_seconds">%1$d m %2$s s</string> <!-- Ex: 2 m 37.93 s -->
```

File: app/src/main/res/values-b+es/strings.xml [hunk 5]
```diff
@@ -109,6 +119,7 @@
     <string name="choose_component">Elegir componente</string>
 
     <string name="pause">Pausar</string>
+    <string name="achievements">Achievements</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:122]:
```suggestion
    <string name="achievements">Logros</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 6]
```diff
@@ -109,6 +119,7 @@
...
     <string name="reset">Resetear</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:123]:
```suggestion
    <string name="reset">Reestablecer</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 7]
```diff
@@ -166,6 +177,8 @@
     <string name="category_video">Video</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:177]:
```suggestion
    <string name="category_video">Vídeo</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 8]
```diff
@@ -166,6 +177,8 @@
...
     <string name="category_video_summary">Filtro, Renderizado de subprocesos, contador de FPS</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:178]:
```suggestion
    <string name="category_video_summary">Filtro, renderizado de subprocesos, contador de FPS</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 9]
```diff
@@ -189,13 +202,20 @@
     <string name="vibrate_on_touch">Vibrar al tacto</string>
     <string name="vibration_strength">Fuerza de vibración</string>
     <string name="soft_input_opacity">Opacidad de entrada suave</string>
-    <string name="cheats">Trucos</string>
-    <string name="cheats_summary">Habilitar e importar códigos de trucos</string>
+    <string name="retroachievements">RetroAchievements</string> <!-- Unir las palabras porque este es el nombre oficial. -->
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:205]:
```suggestion
    <string name="retroachievements">RetroAchievements</string> <!-- Mantén las palabras unidas porque este es el nombre oficial -->
```

File: app/src/main/res/values-b+es/strings.xml [hunk 10]
```diff
@@ -189,13 +202,20 @@
...
+    <string name="retroachievements_summary">Inicio de sesión &amp; Gestionar la conexión de RetroAchievements</string> <!-- Mantener las palabras juntas porque este es el nombre oficial -->
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:206]:
```suggestion
    <string name="retroachievements_summary">Inicio de sesión &amp; Gestionar la conexión de RetroAchievements</string> <!-- Mantén las palabras unidas porque este es el nombre oficial -->
```

File: app/src/main/res/values-b+es/strings.xml [hunk 11]
```diff
@@ -189,13 +202,20 @@
...
+    <string name="enable_rich_presence">Activar la opción para mostrar lo que estás haciendo</string>
+    <string name="rich_presence_summary">Permite que otros usuarios de RetroAchievement vean lo que estás jugando</string>
+    <string name="hardcore_mode">Modo Hardcore</string>
+    <string name="hardcore_mode_summary">Gana el doble de puntos, pero no puedes usar estados de guardado, trucos ni cámara lenta mientras juegas</string>
+
+    <string nam e="cheats">Trucos</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:212]:
```suggestion
    <string name="cheats">Trucos</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 12]
```diff
@@ -189,13 +202,20 @@
...
+    <string name="cheats_summary">Habilitar e importar trucos de &amp; trucos</string>
     <string name="enable_cheats">Habilitar trucos</string>
     <string name="cheats_database">Base de datos de trucos (formato XML)</string>
     <string name="import_cheats">Importar trucos</string>
-    <string name="import_cheats_summary">Importar trucos de archivos base disponibles. Por ahora, solo se admite XML. Se eliminará cualquier truco importado anteriormente</string>
-    <string name="rom_shortcut">Acceso directo de ROM</string>
+    <string name="import_cheats_summary">Importar trucos desde archivos de base de datos para tenerlos disponibles. Por ahora, solo se admiten bases de datos en formato XML. Cualquier truco previamente importado se eliminará</string>
+    <string name="rom_shortcut">Acceso directo a ROM</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:218]:
```suggestion
    <string name="rom_shortcut">Acceso directo a la ROM</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 13]
```diff
@@ -221,6 +241,8 @@
     <string name="label_rom_config_load_gba_rom">Cargar ROM GBA</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:241]:
```suggestion
    <string name="label_rom_config_load_gba_rom">Cargar ROM de GBA</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 14]
```diff
@@ -221,6 +241,8 @@
...
     <string name="label_rom_config_gba_rom_path">Ruta de acceso ROM GBA</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:242]:
```suggestion
    <string name="label_rom_config_gba_rom_path">Ruta de acceso a la ROM de GBA</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 15]
```diff
@@ -221,6 +241,8 @@
...
     <string name="label_rom_config_gba_save_path">Ruta de guardado de GBA</string>
+    <string name="rom_details_configuration_tab">Configuración</string>
+    <string name="retro_achievements_tab">Retro Achievements</string> <!-- Mantenga las palabras separadas para ayudar con el ajuste de palabras -->
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:245]:
```suggestion
    <string name="retro_achievements_tab">Retro Achievements</string> <!-- Mantén las palabras separadas para ayudar con el ajuste de palabras -->
```

File: app/src/main/res/values-b+es/strings.xml [hunk 16]
```diff
@@ -286,8 +308,39 @@
...
-   
-    <string-array name="fast_forward_speed_multiplier_options">
+    
+    <string name="retro_achievements_login_description">¡Inicia sesión con RetroAchievements para ver los logros que puedes desbloquear mientras juegas este juego!</string>
+    <string name="login_with_retro_achievements">Inicia sesión con RetroAchievements</string>
+    <string name="login">Iniciar sesión</string>
+    <string name="retroachievements_login_summary">¡Inicia sesión con RetroAchievements para desbloquear logros mientras juegas!</string>
+    <string name="retroachievements_login_status">Has iniciado sesión como %1$s</string>
+    <string name="retroachievements_logout">Cerrar sesión</string>
+    <string name="retroachievements_logout_confirmation">¿Estás seguro de que deseas cerrar sesión en RetroAchievements? No podrás desbloquear logros mientras juegas.</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:318]:
```suggestion
    <string name="retroachievements_logout_confirmation">¿Estás seguro de que quieres cerrar sesión en RetroAchievements? No podrás desbloquear logros mientras juegas.</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 17]
```diff
@@ -286,8 +308,39 @@
...
+    <string name="retro_achievements_login_error">Hubo un problema al intentar iniciar sesión. Asegúrate de que tu nombre de usuario y contraseña sean correctos y de que estés conectado a Internet</string>
+    <string name="retro_achievements_login_error_short">Se produjo un problema al intentar iniciar sesión</string>
+    <string name="retro_achievements_load_error">Se produjo un problema al intentar cargar los logros de este juego. Asegúrate de estar conectado a Internet e intenta de nuevo más tarde</string>
+    <string name="retro_achievements_no_achievements">No se encontraron RetroAchievements para este juego</string>
+    <string name="retro_achievements_relaunch_to_apply_settings">Reinicia el ROM para aplicar la nueva configuración de RetroAchievements</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:323]:
```suggestion
    <string name="retro_achievements_relaunch_to_apply_settings">Reinicia la ROM para aplicar la nueva configuración de RetroAchievements</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 18]
```diff
@@ -286,8 +308,39 @@
...
+    <string name="username">Nombre de usuario</string>
+    <string name="password">Contraseña</string>
+    <string name="retry">Reintentar</string>
+    <string name="points_abbreviated">PTS</string>
+    <string name="points">Puntos</string>
+    <string name="completed">Completado</string>
+    <string name="ra_mode_hardcore">Hardcore</string>
+    <string name="ra_mode_softcore">Softcore</string>
+    <string name="view_achievement">Ver logro</string>
+    <string name="completed_achievements"><b>%1$d</b> de <b>%2$d</b> (%3$d%%)</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:333]:
```suggestion
    <string name="completed_achievements"><b>%1$d</b> de <b>%2$d</b> (%3$d% %)</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 19]
```diff
@@ -286,8 +308,39 @@
...
+    <string name="submitting_achievements">Enviando RetroAchievements…</string>
+    <string name="achievements_loaded">Achievements loaded</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:335]:
```suggestion
    <string name="achievements_loaded">Logros cargados</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 20]
```diff
@@ -78,9 +84,13 @@
     <string name="cant_load_empty_slot">No se puede cargar un espacio vacío</string>
     <string name="failed_reset_emulation">No se pudo restablecer</string>
     <string name="save_states_not_supported">Los estados de guardado no son compatibles mientras se ejecuta el firmware</string>
-    <string name="rewind_time_seconds">%1$ss</string> <!-- Ex: 43.56s -->
-    <string name="rewind_time_minutes_seconds">%1$dm%2$ss</string> <!-- Ex: 2m37.93s -->
+    <string name="save_states_unavailable_ra_hardcore_enabled">Guardados no disponibles en modo Hardcore</string>
+    <string name="rewind_time_seconds">%1$ss</string> 
+    <string name="rewind_time_minutes_seconds">%1$dm%2$ss</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:89]:
```suggestion
    <string name="rewind_time_seconds">%1$s s</string> <!-- Ej: 43,56 s -->
    <string name="rewind_time_minutes_seconds">%1$d min %2$s s</string> <!-- Ej: 2 min 37,93 s -->
```
?

File: app/src/main/res/values-b+es/strings.xml [hunk 21]
```diff
@@ -286,8 +308,39 @@
...
+    <string name="username">Nombre de usuario</string>
+    <string name="password">Contraseña</string>
+    <string name="retry">Reintentar</string>
+    <string name="points_abbreviated">PTS</string>
+    <string name="points">Puntos</string>
+    <string name="completed">Completado</string>
+    <string name="ra_mode_hardcore">Hardcore</string>
+    <string name="ra_mode_softcore">Softcore</string>
+    <string name="view_achievement">Ver logro</string>
+    <string name="completed_achievements"><b>%1$d</b> de <b>%2$d</b> (%3$d%%)</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:333]:
```suggestion
    <string name="completed_achievements"><b>%1$d</b> de <b>%2$d</b> (%3$d% %)</string>
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-b+es/strings.xml [hunk 22]
```diff
@@ -63,6 +68,7 @@
     <string name="dsiware_manager_import_title_error_not_dsiware_title">No es un título de DSiWare</string>
     <string name="dsiware_manager_import_title_error_title_already_imported">Título de DSiWare ya importado</string>
     <string name="dsiware_manager_import_title_error_insatll_failed">Error al instalar el título</string>
+    <string name="dsiware_manager_import_title_error_metadat_fetch_failed">No se pudo descargar los metadatos del título. Verifique su conexión a internet.</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:71]:
```suggestion
    <string name="dsiware_manager_import_title_error_metadat_fetch_failed">No se pudieton descargar los metadatos del título. Verifica tu conexión a Internet.</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 23]
```diff
@@ -16,6 +16,8 @@
     <string name="undo">Deshacer</string>
     <string name="update">Actualizar</string>
     <string name="close">Cerrar</string>
+    <string name="play">Jugar</string>
+
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:20]:
```suggestion
    <string name="ellipsis">…</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 24]
```diff
@@ -63,6 +68,7 @@
     <string name="dsiware_manager_import_title_error_not_dsiware_title">No es un título de DSiWare</string>
     <string name="dsiware_manager_import_title_error_title_already_imported">Título de DSiWare ya importado</string>
     <string name="dsiware_manager_import_title_error_insatll_failed">Error al instalar el título</string>
+    <string name="dsiware_manager_import_title_error_metadat_fetch_failed">No se pudo descargar los metadatos del título. Verifique su conexión a internet.</string>
     <string name="dsiware_manager_import_title_error_unknown">Se ha producido un error desconocido</string>
     <string name="dsiware_import_from_file">Desde el archivo</string>
     <string name="dsiware_import_from_rom_list">De la lista de ROM</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:74]:
```suggestion
    <string name="dsiware_import_from_rom_list">Desde la lista de ROM</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 25]
```diff
@@ -78,9 +84,13 @@
     <string name="cant_load_empty_slot">No se puede cargar un espacio vacío</string>
     <string name="failed_reset_emulation">No se pudo restablecer</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:85]:
```suggestion
    <string name="failed_reset_emulation">No se pudo restablecer la emulación</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 26]
```diff
@@ -78,9 +84,13 @@
...
     <string name="save_states_not_supported">Los estados de guardado no son compatibles mientras se ejecuta el firmware</string>
-    <string name="rewind_time_seconds">%1$ss</string> <!-- Ex: 43.56s -->
-    <string name="rewind_time_minutes_seconds">%1$dm%2$ss</string> <!-- Ex: 2m37.93s -->
+    <string name="save_states_unavailable_ra_hardcore_enabled">Guardados no disponibles en modo Hardcore</string>
+    <string name="rewind_time_seconds">%1$ss</string> 
+    <string name="rewind_time_minutes_seconds">%1$dm%2$ss</string> 
     <string name="rewind_not_enabled">El rebobinado no está habilitado</string>
+    <string name="rewind_unavailable_ra_hardcore_enabled">Rebobinado no disponible en modo Hardcore</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:91]:
```suggestion
    <string name="rewind_unavailable_ra_hardcore_enabled">El rebobinado no disponible en modo Hardcore</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 27]
```diff
@@ -286,8 +308,39 @@
     <string name="background_deleted">Fondo eliminado</string>
     <string name="background_add_wrong_orientation">Se agregó fondo pero no es visible para la orientación seleccionada.</string>
     <string name="background_add_processing_failed">Error al procesar la imagen de fondo seleccionada. Asegúrese de que no esté dañado.</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:310]:
```suggestion
    <string name="background_add_processing_failed">Error al procesar la imagen de fondo seleccionada. Asegúrate de que no esté dañada.</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 28]
```diff
@@ -286,8 +308,39 @@
...
-   
-    <string-array name="fast_forward_speed_multiplier_options">
+    
+    <string name="retro_achievements_login_description">¡Inicia sesión con RetroAchievements para ver los logros que puedes desbloquear mientras juegas este juego!</string>
+    <string name="login_with_retro_achievements">Inicia sesión con RetroAchievements</string>
+    <string name="login">Iniciar sesión</string>
+    <string name="retroachievements_login_summary">¡Inicia sesión con RetroAchievements para desbloquear logros mientras juegas!</string>
+    <string name="retroachievements_login_status">Has iniciado sesión como %1$s</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:316]:
```suggestion
    <string name="retroachievements_login_status">Sesión iniciada como %1$s</string>
```

File: app/src/main/res/values-b+es/strings.xml [hunk 29]
```diff
@@ -286,8 +308,39 @@
...
+    <string name="achievements_unlocked_compact">Desbloqueados: %1$d/%2$d</string>
+    <<string name="achievements_failed_load">¡Error al cargar los logros!</string>
+    <string name="achievements_failed_load_tip">Asegúrate de estar en línea</string>
+    <string name="achievement_unlocked">¡Logro desbloqueado!</string>
+    <string name="achievement_missable">Logro perdible</string>
+    <string name="achievement_missable_description">Este logro se puede perder durante una partida</string>
+       
+       <string-array name="fast_forward_speed_multiplier_options">
         <item>Ilimitado</item>
         <item>1.5x</item>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:345]:
```suggestion
        <item>1,5x</item>
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

**Comment by BackpackXl** [comment]
Thank you for improving and fixing!"

File: app/src/main/res/values-b+es/strings.xml [hunk 30]
```diff
@@ -63,9 +68,10 @@
     <string name="dsiware_manager_import_title_error_not_dsiware_title">No es un título de DSiWare</string>
     <string name="dsiware_manager_import_title_error_title_already_imported">Título de DSiWare ya importado</string>
     <string name="dsiware_manager_import_title_error_insatll_failed">Error al instalar el título</string>
+    <string name="dsiware_manager_import_title_error_metadat_fetch_failed">No se pudieton descargar los metadatos del título. Verifica tu conexión a Internet.</string>
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:71]: my typo
```suggestion
    <string name="dsiware_manager_import_title_error_metadat_fetch_failed">No se pudieron descargar los metadatos del título. Verifica tu conexión a Internet.</string>
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-b+es/strings.xml [hunk 31]
```diff
@@ -76,11 +82,15 @@
...
     <string name="failed_load_state">No se pudo cargar</string>
     <string name="cant_load_empty_slot">No se puede cargar un espacio vacío</string>
-    <string name="failed_reset_emulation">No se pudo restablecer</string>
+    <string name="failed_reset_emulation">No se pudo restablecer la emulación</string>
     <string name="save_states_not_supported">Los estados de guardado no son compatibles mientras se ejecuta el firmware</string>
-    <string name="rewind_time_seconds">%1$ss</string> <!-- Ex: 43.56s -->
-    <string name="rewind_time_minutes_seconds">%1$dm%2$ss</string> <!-- Ex: 2m37.93s -->
+    <string name="save_states_unavailable_ra_hardcore_enabled">Guardados no disponibles en modo Hardcore</string>
+    <string name="rewind_time_seconds">%1$s s</string> <!-- Ej: 43,56 s -->
+    <string name="rewind_time_minutes_seconds">%1$d min %2$s s</string> <!-- Ej: 2 min 37,93 s -->
```
Comment (agatemosu) [inline app/src/main/res/values-b+es/strings.xml:89]: sorry
```suggestion
    <string name="rewind_time_seconds">%1$s s</string> <!-- Ej: 43,56 s -->
    <string name="rewind_time_minutes_seconds">%1$d min %2$s s</string> <!-- Ej: 2 min 37,93 s -->
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

**Review by agatemosu — APPROVED** [review]

**Comment by rafaelvcaetano** [comment]
Thank you both for you amazing effort 🙂

**Pull request merged**